app/Controllers/indexController.php

```diff
@@ -259,6 +259,23 @@ public function aboutAction() {
 		Minz_View::prependTitle(_t('index.about.title') . ' · ');
 	}
 
+	/**
+	 * This action displays the EULA page of FreshRSS.
+	 * This page is enabled only if admin created a data/tos.html file.
+	 * The content of the page is the content of data/tos.html.
+	 * It returns 404 if there is no EULA.
+	 */
+	public function tosAction() {
+		$tos_filepath = DATA_PATH . '/tos.html';
+		if (!file_exists($tos_filepath)) {
```

Comment:
I suggest to test for the return value of file_get_contents() instead of an additional test (which is less robust for e.g. access rights and race conditions)